clang/cmake/caches/llvm-toolchain/stage1.cmake

```
@@ -0,0 +1,17 @@
+# Stage 1
+# * Build an LTO optimized libcxx, so we can staticially link it into stage 2
+#   clang.
+
+
+set(CMAKE_BUILD_TYPE Release CACHE STRING "")
+set(LLVM_TARGETS_TO_BUILD Native CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt;libcxx;libcxxabi;libunwind" CACHE STRING "" FORCE)
```

This is a nit, but I find it cleaner to write the list out instead of using semicolons manually (which also allows you to split it over multiple lines if needed). Also, is the FORCE required?
```suggestion
set(LLVM_ENABLE_RUNTIMES compiler-rt libcxx libcxxabi libunwind CACHE STRING "" FORCE)
```

---

FORCE isn't required in the stage1 spec, so I'll remove it.